addons/auth/ember-cli-build.js

```diff
@@ -18,6 +17,7 @@ module.exports = function (defaults) {
     behave. You most likely want to be modifying `./index.js` or app's build file
   */
 
+  const { maybeEmbroider } = require('@embroider/test-setup');
```

Comment:
For readibility, I do preffer the import of `maybeEmbroider` at the top of the document as we do in all of them.

Reply:
I'm fine with that.  I was just doing a strict transfer from a brand new ember addon.

Reply:
Yep, the upgrade script or creating an addon from scratch adds it in the middle of the document, being there done that 😅